.misc/deploy.pypi.sh

```diff
@@ -11,11 +11,17 @@ echo "Uploading coala to pypi"
 pip3 install twine wheel
 python3 setup.py sdist bdist_wheel
 # Upload one by one to avoid timeout
-# Don't block merging because pypi is in error 500 mood again
-set +e
-twine upload dist/*.whl -u "$PYPIUSER" -p "$PYPIPW"
-twine upload dist/*.tar.gz -u "$PYPIUSER" -p "$PYPIPW"
-set -e
+twine upload dist/* -u "$PYPIUSER" -p "$PYPIPW" 2>&1 | tee er.txt
+if [ "${PIPESTATUS[0]}" -eq 0 ]; then
+   echo "Uploaded wheel and tar to pypi"
+else
+    ERROR=$(<er.txt)
```

Comment:
Instead of reading the file into a variable in bash, and then using bash limited text tools, just use `grep -q` on the file and check the exit status of grep.